docs/docs/workflows/terraform-module.md

```diff
@@ -0,0 +1,156 @@
+        disk_size     = 30
+        disk_type     = "Premium_LRS"
+        initial_count = 2
+      }
+    }
+  }
+  ```
+
+  </tabItem>
+
+  <tabItem value="aws" label="AWS">
+
+  ```
+  locals {
+    module_version = "v0.0.0" // insert Constellation version here.
+  }
+
+  module "aws-constellation" {
+    source = "https://github.com/edgelesssys/constellation/releases/download/${local.module_version}/terraform-module.zip//modules/aws-constellation"
+    name        = "constell"
+    zone        = "us-east-2c"
+    name_prefix = "example"
+    node_groups = {
+      control_plane_default = {
+        role          = "control-plane"
+        zone          = "us-east-2c"
+        instance_type = "m6a.xlarge"
+        disk_size     = 30
+        disk_type     = "gp3"
+        initial_count = 3
+      },
+      worker_default = {
+        role          = "worker"
+        zone          = "us-east-2c"
+        instance_type = "m6a.xlarge"
+        disk_size     = 30
+        disk_type     = "gp3"
+        initial_count = 2
+      }
+    }
+  }
+  ```
+
+  </tabItem>
+
+  <tabItem value="gcp" label="GCP">
+
+  ```
+  locals {
+    module_version = "v0.0.0" // insert Constellation version here.
+  }
+
+  module "gcp-constellation" {
+    source = "https://github.com/edgelesssys/constellation/releases/download/${local.module_version}/terraform-module.zip//modules/gcp-constellation"
+    name    = "constell"
+    project = "constell-proj" // replace with your project id
+    service_account_id = "constid"
+    zone    = "europe-west2-a"
+    node_groups = {
+      control_plane_default = {
+        role          = "control-plane"
+        zone          = "europe-west2-a"
+        instance_type = "n2d-standard-4"
+        disk_size     = 30
+        disk_type     = "pd-ssd"
+        initial_count = 3
+      },
+      worker_default = {
+        role          = "worker"
+        zone          = "europe-west2-a"
+        instance_type = "n2d-standard-4"
+        disk_size     = 30
+        disk_type     = "pd-ssd"
+        initial_count = 2
+      }
+    }
+  }
+  ```
+
+  </tabItem>
+  </tabs>
+
+3. Initialize and apply the module.
+  ```bash
+  terraform init
+  terraform apply
+  ```
+
+## Custom setup
+If you need to separate IAM and cluster management or need custom infrastructure, you can also call the submodules individually.
+You can have a look at the respective convenience module (`{csp}-constellation`) for how the module calls can be structured.
+The submodules are:
+- `constellation-cluster`: manages the Constellation cluster
+- `fetch-image`: translates the Constellation image version to the image ID of the cloud service provider
+- `infrastructure/{csp}`: contains the cluster infrastructure resources
+- `infrastructure/iam/{csp}`: contains the IAM resources used within the cluster
+
+## Cluster upgrades
+:::tip
+For general information on cluster upgrades, please refer to [Upgrade your cluster](./upgrade.md).
```

Comment:
```suggestion
For general information on cluster upgrades, please refer to the specific documentation for [Upgrading your cluster](./upgrade.md).
```

I don't know if this is perfect, but otherwise the sentence doesn't make sense imo.